redbot/cogs/warnings/warnings.py

```diff
@@ -24,7 +25,14 @@
 class Warnings(commands.Cog):
     """Warn misbehaving users and take automated actions."""
 
-    default_guild = {"actions": [], "reasons": {}, "allow_custom_reasons": False}
+    default_guild = {
+        "actions": [],
+        "reasons": {},
+        "allow_custom_reasons": False,
+        "toggle_dm": True,
```

Comment:
IMO, this should default to False so that guild owners/admins can choose whether or not for this to be enabled.

Reply:
I agree, though warnings cog was acting as if it was set to `True`, so this would change that behavior and it should be included included in changelog, if that's going to be changed to `False` as default.